### ISSUE TITLE
[Accessibility] Check placeholder text contrast in forms

### ISSUE THREAD

**Issue by jaredhenderson** [unimelb/pattern-lib#1272]
## Expected behaviour

Text should meet normal 4.5:1 contrast requirements.

## Current behaviour

Probably doesn't meet requirements at present.

**Comment by jaredhenderson** [comment]
Moved to JIRA
https://jira.unimelb.edu.au/browse/TS-1201
